sirepo/package_data/static/json/radia-schema.json

```diff
@@ -161,7 +161,8 @@
             "precision": ["Precision (T)", "Float", 0.00010, "", ""],
             "result": ["Result", "String", ""],
             "sbatchCores": ["Cores", "Integer", 1],
-            "sbatchHours": ["Hours", "Float", 0.4]
+            "sbatchHours": ["Hours", "Float", 0.4],
+            "sbatchQueue": ["Queue", "SbatchQueue", "debug"]
```

Comment:
SbatchQueue => NERSCQueue, because that's what it is for now. Leave sbatchQueue as the field, since that may be used if we add another HPC with multiple queues. Then, we'll have to solve the 2-case. The word NERSCQueue here flags it as a 1-case.

Reply:
https://github.com/radiasoft/sirepo/pull/2511/commits/779ddedc7bc17f3c385612be4fee7c6b6a503204